app/build.gradle.kts

```diff
@@ -111,6 +111,10 @@ dependencies {
     implementation(Libs.Android.activityCompose)
 
     testImplementation(Libs.Test.junit)
+    testImplementation(Libs.Test.mockkAndroid)
+    testImplementation(Libs.Test.mockkAgent)
+    testImplementation ("org.jetbrains.kotlinx:kotlinx-coroutines-test:1.6.4")
```

Comment:
Please put the dependency in the Libs.Test.. thingie

Reply:
@levinzonr I've made the necessary change. Please review it again.